lib/orderbook/OrderBook.ts

```diff
@@ -289,25 +289,37 @@ class OrderBook extends EventEmitter {
     // perform match. maker orders will be removed from the repository
     const tp = this.getTradingPair(order.pairId);
     const matchingResult = tp.match(order);
```

Comment:
An empty line after this one would help. Also IMO we need now to add very verbose documentation for this method (`placeOrder`).

Reply:
I added this line and some comments in a separate commit, let me know if you think it's sufficient or if you have anything else in mind.

Reply:
Looks great, thanks.